src/connection.rs

```diff
@@ -173,14 +181,26 @@ mod test {
 
     #[test]
     fn local_connection() {
-
         Connection::recreate_local("test.fdb".to_string())
             .expect("Error on recreate the test database");
 
         let conn = Connection::open_local("test.fdb".to_string())
             .expect("Error on connect the test database");
 
-        conn.close()
-            .expect("error on close the connection");
+        conn.close().expect("error on close the connection");
+    }
+
+    #[test]
+    fn remote_connection() {
+        let conn = Connection::open(
+            "localhost".into(),
+            3050,
+            "/var/lib/firebird/system/test.fdb".into(),
```

Comment:
I think we can't use a full path file into tests or examples.

We can use paths like "test.fdb" with the "DatabaseAccess = Restrict" setting into firebird.conf(https://firebirdsql.org/rlsnotesh/config-fb-conf.html)

Reply:
Oh, thanks for the info, i will fix it